### PR TITLE
Add documentation to the help tab

### PR DESCRIPTION
**Pull request by DannyDakota**

### CODE REVIEW

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -15,8 +15,67 @@
...
+        + "For a full comprehensive guide, please refer to " + USERGUIDE_URL
+        + "\n"
+
+        + "\n1. FIND a particular contact in Tracey."
+        + NEWLINE_AND_INDENTATION
+        + "Format: find NAME"
+        + NEWLINE_AND_INDENTATION
+        + "Example 1: find John\n"
+
+        + "\n2. ADD a student with relevant details into Tracey."
```
Comment (cwq2326) [inline src/main/java/seedu/address/ui/HelpWindow.java:32]: Maybe can provide the user with the list of possible tags

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 2]
```diff
@@ -15,8 +15,67 @@
...
+        + "\n4. EDIT the person at the specified INDEX. The index refers to the index number shown in the " +
+            "displayed person list."
+        + NEWLINE_AND_INDENTATION
+        + "Format: edit INDEX n/NAME ..."
+        + NEWLINE_AND_INDENTATION
+        + "Example 1: edit 1 p/91234567 e/johndoe@example.com"
+        + NEWLINE_AND_INDENTATION
+        + "Example 2: edit 2 n/David Limpeh t/\n"
+
+        + "\n5. CLEAR all the data inside Tracey"
```
Comment (cwq2326) [inline src/main/java/seedu/address/ui/HelpWindow.java:58]: Maybe can include a confirmation pop-up will appear or a warning that says cleared data cannot be retrieved

**Review by cwq2326 — COMMENTED** [review]
Just some minor changes to add command and clear command

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (DannyDakota) [reply src/main/java/seedu/address/ui/HelpWindow.java:32]: Noted! I think this is a good idea. I have added this in! 👍

**Pull request closed without merging**